openapi/components/schemas/PersonalDetailsRegistration.yml

```diff
@@ -0,0 +1,8 @@
+type: object
+properties:
+  gmcNumber:
+    type: string
+    pattern: \d{7}
+  gdcNumber:
+    type: string
+    pattern: D\d{6}
```

Comment:
There is also PublicHealth number I think - assuming we can ignore this for now

Reply:
GDC isn't even on the design, so I'm already one step ahead 😄